packages/dd-appsec/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
```

Comment:
Let's put this in dd-trace until we figure out the actual module structure, especially since right now it will be impossible to fully isolate it.